centreon/src/Core/HostCategory/Infrastructure/API/FindHostCategories/FindHostCategoriesRoute.yaml

```diff
@@ -0,0 +1,5 @@
+findHostCategories:
```

Comment:
Laurent created route names in `PascalCase`, I did too, maybe you should do the same ?